backend/services/BroadcastService.ts

```diff
@@ -420,6 +420,25 @@ const isBroadcastRunning = async (): Promise<boolean> => {
   return jobs.some((job: { jobname: string }) => job.jobname != 'invoke-broadcast' && JOB_NAMES.includes(job.jobname))
 }
 
+const updateSubscriptionStatus = async(
+  phoneNumber: string,
+  isUnsubscribe: boolean,
+  authorName: string,
+) => {
+  await supabase
+    .update(authors)
+    .set({ unsubscribed: isUnsubscribe })
+    .where(eq(authors.phoneNumber, phoneNumber));
+
+  let postMessage = ''
+  if (isUnsubscribe) {
+    postMessage = `This phone number ${phoneNumber} has now been unsubscribed by ${authorName}.`;
+  } else {
+    postMessage = `This phone number ${phoneNumber} has now been resubscribed by ${authorName}.`;
+  }
+  return await MissiveUtils.createPost(postMessage);
```

Comment:
It should handle the response result here.

I think it's best for MissiveUtils to log the response body for debugging later.